server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -35,33 +33,25 @@ public class GetSnapshotsResponse extends ActionResponse implements ChunkedToXCo
 
     private final List<SnapshotInfo> snapshots;
 
-    @UpdateForV9(owner = UpdateForV9.Owner.DISTRIBUTED_COORDINATION) // always empty, can be dropped
-    private final Map<String, ElasticsearchException> failures;
-
     @Nullable
     private final String next;
 
     private final int total;
 
     private final int remaining;
 
-    public GetSnapshotsResponse(
-        List<SnapshotInfo> snapshots,
-        Map<String, ElasticsearchException> failures,
-        @Nullable String next,
-        final int total,
-        final int remaining
-    ) {
+    public GetSnapshotsResponse(List<SnapshotInfo> snapshots, @Nullable String next, final int total, final int remaining) {
         this.snapshots = List.copyOf(snapshots);
-        this.failures = failures == null ? Map.of() : Map.copyOf(failures);
         this.next = next;
         this.total = total;
         this.remaining = remaining;
     }
 
     public GetSnapshotsResponse(StreamInput in) throws IOException {
         this.snapshots = in.readCollectionAsImmutableList(SnapshotInfo::readFrom);
-        this.failures = Collections.unmodifiableMap(in.readMap(StreamInput::readException));
+        if (in.getTransportVersion().before(TransportVersions.REMOVE_SNAPSHOT_FAILURES)) {
+            in.readMap(StreamInput::readException);
```

Comment:
Could you add a comment saying what this field used to be for?

Reply:
Added in [a748d09](https://github.com/elastic/elasticsearch/pull/114496/commits/a748d093ea6b22a75e9a676aec845c88d0f73446)